### DISCUSSION TITLE
Jump to related record

### DISCUSSION THREAD

**Discussion by crudyrudy** [Budibase/budibase#2460] [Ideas]
Hello together!
This is the first discussion I start in this forum. Therefore, I would first like to thank the creators and supporters of this very promising low code platform. Great work!

And here is my first suggestion:

In the table component, related records are displayed like buttons. In the backend they also behave like buttons: A click brings up a list of related records. Unfortunately, this is not the case in the frontend. So far I have not found a way to implement this function in the frontend. This feature would be very useful, I think. What do you think?

**Comment by KernOil** [comment]
I agree with you @crudyrudy I think this would be a great feature. I would use this for sure!

**Comment by joebudi** [comment]
Hey!

Thanks for your kind words @crudyrudy .
Would the following work as a solution:
Click a table row, it opens a modal presenting the records details, whilst also listing the record's related records?

**Reply by KernOil** [reply]
@joebudi  that is the way I have it set right now with a button. but would love to see the related table when selecting the related component and would love in table edits vs having to go to a separate page and submit a new form!

**Reply by crudyrudy** [reply]
@joebudi That would be great! Is this already possible?

**Reply by crudyrudy** [reply]
@AndrewJRamay How can I create a modal?

**Reply by mjashanks** [reply]
This is coming in the next release, in just a few days!

**Reply by crudyrudy** [reply]
Perfect!
